parser_tudi.py

```diff
@@ -190,71 +230,180 @@ def p_type_dims(p):
     elif len(p) == 4:
         p[0] = 'arr1d'
 
-# Pending to add unary logic operator NOT
-def p_logicop(p):
-    '''logicop : AND
-               | OR'''
 
 def p_god_exp(p):
-    '''god_exp : super_exp god_exp_prima'''
+    '''god_exp : super_exp god_exp_neuro_1 god_exp_prima'''
+
+def p_god_exp_neuro_1(p):
+    '''god_exp_neuro_1 : '''
+    if len(operator_stack) > 0:
+        if operator_stack[-1] in arr_logicops:
+            right_oper = operand_stack.pop()
```

Comment:
Unir con un `and` pa' ahorrarnos la _indentación_. (Mismo comentario en los otros 3 que checan el length del stack)